core/src/types/operator/operator.rs

```diff
@@ -729,7 +729,8 @@ impl Operator {
                     }
 
                     let (_, mut w) = inner.write(&path, args).await?;
-                    w.write(bs).await?;
+                    // FIXME: we should bench here to measure the perf.
```

Comment:
Maywe open a new experiment PR to bench it?

Reply:
> Maywe open a new experiment PR to bench it?

Yep, we will raise a new PR to measure and improve it.